templates/events-header.php

```diff
@@ -60,6 +60,8 @@
 			</span>
 			<?php if ( current_user_can( 'edit_translation_event', $event->id() ) ) : ?>
 				<a class="event-page-edit-link" href="<?php echo esc_url( Urls::event_edit( $event->id() ) ); ?>"><span class="dashicons dashicons-edit"></span><?php esc_html_e( 'Edit event', 'gp-translation-events' ); ?></a>
+			<?php elseif ( current_user_can( 'manage_translation_event_attendees', $event->id() ) ) : ?>
```

Comment:
This also shows on the Manage Attendees page itself which is redundant:
![Screenshot 2024-05-07 at 10 58 32](https://github.com/WordPress/wporg-gp-translation-events/assets/203408/49e18cc5-7337-43cf-a5c6-2dca9fd13f59)

Reply:
`Manage attendees` link removed.

<img width="1476" alt="image" src="https://github.com/WordPress/wporg-gp-translation-events/assets/2834132/e2d87deb-494e-427b-978f-ecea350ca491">